src/impact-dashboard/EpidemicModelContext.tsx

```diff
@@ -82,8 +78,8 @@ export type EpidemicModelPersistent = ModelInputsPersistent &
 // TODO: is there a smarter way to get these values?
 export const persistedKeys: Array<keyof EpidemicModelPersistent> = [
   "countyName",
-  "facilityName",
   "stateCode",
+  "hospitalBeds",
```

Comment:
Turns out we do want to persist `hospitalBeds`.  If we don't, the app seems to be loading the "US Total" numbers and breaks the charts as follows:

<img width="833" alt="Screen Shot 2020-04-18 at 10 11 26 AM" src="https://user-images.githubusercontent.com/25503/79642633-c59be200-816c-11ea-9c63-456a0b89ccb8.png">

Reply:
CC'ing @macfarlandian just in case this is surprising to him for any reason.

Reply:
FWIW I think storing hospital beds is a mistake. The value is not user-editable, but it is fetched from the google sheet as "locale data". so the value we would store in firestore is at best redundant and at worst out of date. If the facility is associated with a state and county but they aren't being properly selected by this view to populate this data, then that is an issue we should fix